experimenter/experimenter/experiments/constants.py

```diff
@@ -212,7 +212,8 @@ class Application(models.TextChoices):
         APPLICATION_CONFIG_MONITOR_WEB.slug,
         APPLICATION_CONFIG_MONITOR_WEB.name,
     )
-    DEMO_APP = (APPLICATION_CONFIG_DEMO_APP.slug, APPLICATION_CONFIG_DEMO_APP.name)
+    DEMO_APP = (APPLICATION_CONFIG_DEMO_APP.slug,
+                APPLICATION_CONFIG_DEMO_APP.name)
```

Comment:
Was this change needed? Maybe you were using a formatter or something

Reply:
I was using a formatter, yes, and it aligned it to the format used with other lines, but I can revert it to the original formatting.